src/index.js

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
-const sessionId =
-  window.localStorage.getItem('_gitclubGameSession') ||
-  [...Array(30)].map(() => Math.random().toString(36)[2]).join('');
-console.log(sessionId);
-window.localStorage.setItem('_gitclubGameSession', sessionId);
-
 const components = require('./components');
 const levels = require('../levels');
 const vanillatoasts = require('vanillatoasts');
 
+const sessionId = window.localStorage.getItem('_gitclubGameSession')
+  || [...Array(30)].map(() => Math.random().toString(36)[2]).join('');
+console.log(sessionId);
```

Comment:
Is logging this to the console intentional?

Reply:
Removed in 27f8c59